src/app/child-dev-project/children/dashboard-widgets/entity-count-dashboard/entity-count-dashboard.component.ts

```diff
@@ -1,46 +1,48 @@
-import { Component, Input, OnInit } from "@angular/core";
+import { Component, OnInit } from "@angular/core";
 import { Router } from "@angular/router";
 import { OnInitDynamicComponent } from "../../../../core/view/dynamic-components/on-init-dynamic-component.interface";
 import { ConfigurableEnumValue } from "../../../../core/configurable-enum/configurable-enum.interface";
 import { Child } from "../../model/child";
 import { DynamicComponent } from "../../../../core/view/dynamic-components/dynamic-component.decorator";
 import { EntityMapperService } from "../../../../core/entity/entity-mapper.service";
+import { Entity } from "../../../../core/entity/model/entity";
 
 @DynamicComponent("ChildrenCountDashboard")
+@DynamicComponent("EntityCountDashboard")
 @Component({
-  selector: "app-children-count-dashboard",
-  templateUrl: "./children-count-dashboard.component.html",
-  styleUrls: ["./children-count-dashboard.component.scss"],
+  selector: "app-entity-count-dashboard",
+  templateUrl: "./entity-count-dashboard.component.html",
+  styleUrls: ["./entity-count-dashboard.component.scss"],
 })
-export class ChildrenCountDashboardComponent
+export class EntityCountDashboardComponent
   implements OnInitDynamicComponent, OnInit
 {
+  private entity = Child.ENTITY_TYPE;
   /**
    * The property of the Child entities to group counts by.
    *
    * Default is "center".
    */
-  @Input() groupBy: string = "center";
+  private groupBy = "center";
```

Comment:
any particular reason to remove this as `@Input `?

I still like the idea that generally components can be configured equivalently through input or config.

Reply:
I remove it because we are not using it and I am also not sure what the support is (whether it still works). Other dashboards also don't allow it (newer ones have never been built this way). So consistency wise I find it a bit difficult to keep it here (why is it in one dashboard and not in others?). But of course not critical, just me trying to reduce things.

Reply:
my longer-term goal would still be that we do away with the onInitDynamic and let the system set config properties automatically to any input property and afterwards trigger the normal ngOnInit. I had gotten rather far with this for most components (like the dashboards, I think) and only got stuck on a few interlinked ones.

But we can re-introduce the @Input annotations also when we implement that.